src/lib/Command/IbexaSetupCommand.php

```diff
@@ -172,7 +172,20 @@ static function (SplFileInfo $dir): string {
             )
         );
 
-        $productPackage = InstalledVersions::getRawData()['versions'][$product];
+        $allRawDataSets = InstalledVersions::getAllRawData();
+        $productPackage = null;
+
+        foreach ($allRawDataSets as $rawData) {
+            if (isset($rawData['versions'][$product])) {
+                $productPackage = $rawData['versions'][$product];
+                break;
+            }
+        }
+
+        if ($productPackage === null) {
+            throw new RuntimeException(sprintf('Could not find product "%s" in installed versions.', $product));
+        }
+
```

Comment:
What is the reason behind this change ?

Reply:
https://github.com/composer/composer/blob/main/src/Composer/InstalledVersions.php#L269
Due to this, when script called InstalledVersions::getRawData()['versions'][$product] I got 'no such key in array' error